contributors.json

```diff
@@ -663,4 +663,14 @@
 			"YARG": [ "Developer" ]
 		}
 	}
+		{
```

Comment:
You have an extra tab here, and you're missing the comma at the end of the previous line.